code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -33,6 +39,66 @@ obj/item/device/mmi/Destroy()
 
 	// Return true if handled
 /obj/item/device/mmi/proc/try_handling_mommi_construction(var/obj/item/O as obj, var/mob/user as mob)
```

Comment:
Why are you doing this in mommi construction?

Reply:
That flew in 2016 /17. I did put most of it in mommi subtypes after I realized that was there.